src/explorer/explorerTreeController.ts

```diff
@@ -82,11 +84,29 @@ export default class ExplorerTreeController implements vscode.TreeDataProvider<v
 
     treeView.onDidChangeSelection((event: any) => {
       if (event.selection && event.selection.length === 1) {
-        if (event.selection[0].isShowMoreItem) {
-          event.selection[0].onShowMoreClicked();
+        const selectedItem = event.selection[0];
+        if (selectedItem.isShowMoreItem) {
```

Comment:
Can we have `a rule` when we use empty lines? :) eg empty line after variables declaration, or around if/else statements. It is not necessary at all! Just some thoughts of a perfectionist :))

Reply:
Sure, so after variable statements and before `if` statements it would be preferable that there's an empty line? Fixed here - I think I sometimes don't leave an empty space when the variable we're setting is exclusively used in the if statements. I think empty space looks better now that it's added tho.